core/src/main/java/com/klaytn/caver/rpc/Klay.java

```diff
@@ -0,0 +1,1295 @@
+     * It set "isFullTransaction" param to true.
+     * @param blockHash The hash of block.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByHash(String blockHash) {
+        return getBlockByHash(blockHash, true);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * @param blockHash The hash of block.
+     * @param isFullTransaction If true it returns the full transaction objects, if false only the hashes of the transactions.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByHash(String blockHash, boolean isFullTransaction) {
+        return new Request<>(
+                "klay_getBlockByHash",
+                Arrays.asList(blockHash, isFullTransaction),
+                web3jService,
+                KlayBlock.class);
+    }
+
+    /**
+     * Returns receipts included in a block identified by block hash.
+     * @param blockHash The hash of block.
+     * @return BlockReceipt
+     */
+    public Request<?, BlockReceipts> getBlockReceipts(String blockHash) {
+        return new Request<>(
+                "klay_getBlockReceipts",
+                Arrays.asList(blockHash),
+                web3jService,
+                BlockReceipts.class);
+    }
+
+    /**
+     * Returns the number of transactions in a block matching the given block number.
+     * @param blockNumber The block number.
+     * @return Quantity
+     */
+    public Request<?, Quantity> getTransactionCountByNumber(long blockNumber) {
+        DefaultBlockParameterNumber blockParameterNumber = new DefaultBlockParameterNumber(blockNumber);
+        return getTransactionCountByNumber(blockParameterNumber);
+    }
+
+    /**
+     * Returns the number of transactions in a block matching the given block number.
+     * @param blockTag The string "latest", "earliest" or "pending"
+     * @return Quantity
+     */
+    public Request<?, Quantity> getTransactionCountByNumber(DefaultBlockParameter blockTag) {
+        return new Request<>(
+                "klay_getBlockTransactionCountByNumber",
+                Arrays.asList(blockTag),
+                web3jService,
+                Quantity.class);
+    }
+
+    /**
+     * Returns the number of transactions in a block from a block matching the given block hash.
+     * @param blockHash The hash of a block
+     * @return Quantity
+     */
+    public Request<?, Quantity> getTransactionCountByHash(String blockHash) {
+        return new Request<>(
+                "klay_getBlockTransactionCountByHash",
+                Arrays.asList(blockHash),
+                web3jService,
+                Quantity.class);
+    }
+
+    /**
+     * Returns a block with consensus information matched by the given hash.
+     * @param blockHash The hash of a block.
+     * @return KlayBlockWithConsensusInfo
+     */
+    public Request<?, KlayBlockWithConsensusInfo> getBlockWithConsensusInfoByHash(String blockHash) {
```

Comment:
This also use old `KlayTransactionReceipt.TransactionReceipt`

Reply:
I added BlockWithConsensusInfo class.